style.css

```diff
@@ -0,0 +1,119 @@
+*{
+    margin: 0;
+    padding: 0;
+}
+header{
+    display: flex;
+    margin: auto;
+    margin-bottom: -0.8rem;
+    justify-content: space-between;
+    padding: 2rem 2rem 0 139px;
+}
+.heading{
+    display: flex;
+   
+}
+.head a{
+    margin-left: 3rem;
+    color: black;
+    text-decoration: none;
+    font-size: 20px;
+    font-family: 'Roboto', sans-serif;
+}
+.head a:hover{
+    text-decoration: underline blue;
+    color: blue;
+}
+.logo{
+    background-image: url(jambito.png);
+    height: 60px;
+    width: 60px;
+    background-size: contain;
+    background-repeat: no-repeat;  
+}
+button{
+    padding: 0.7rem;
+    width: 7rem;
+    margin-top: -12px;
+    background-color: white;
+    border: 1px solid blue;

```

Comment:
add cursor pointer

Reply:
increase the border radius on the buttons

Reply:
had a blue background to the login button and a white text when it is hovered upon